### PR TITLE
Addtion of accessibility for Portfolio Webiste

### PR DESCRIPTION
**Pull request by umair-rao**
I have added accessibility to the portfolio website covering and explaining all below mentioned points.

List all points that have been checked:

- Page titles (Added)
- Image text alternatives (Added)
- Text headings (Added)
- Color contrast (Contrast are as per Figma Template)
- Resize (Not required because the text is clearly readable) 
- Interaction (Added)
- Moving content ( Not Required as the website doesn't have any moving content)
- Multimedia ( Not Required as the website doesn't have any Multimedia)

Please review my recent changes as described above in this PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
         </nav>
         </header>
         <section class="headline_section">
-        <img id="headline-img-mobile" src="images/header-illsutration-mobile.svg" alt="background_image">
-        <img id="headline-img-desktop" src="images/Header-llustration-desktop.svg" alt="background_image">
+        <img id="headline-img-mobile" src="images/header-illsutration-mobile.svg" alt="Mobile header background image">
+        <img id="headline-img-desktop" src="images/Header-llustration-desktop.svg" alt="Desktop header background image">
         <div class="text">
         <div class="primary-text">
             <span id="first">Hey there. I’m Umair Muqeem</span><br><span id="second">I’m a software developer</span>
```
Comment (Qoosim) [inline index.html]: - [ ] This portion of your page serves as your title and the best tag for the first heading on a page. I would recommend you to kindly change the `<span>` tag to `<h1>` as you don't have any on the page already.

File: index.html [hunk 2]
```diff
@@ -41,12 +41,12 @@
...
                 <li><a href=""><img src="images/medium.svg" alt="Mediumlogo" /></a></li>
-                <li><a href="https://github.com/umair-rao" target="_blank"><img src="images/github.svg" alt="github logo" /></a></li>
-                <li><a href=""><img src="images/angellist.svg" alt="angellist logo" /></a></li>
+                <li><a href="https://github.com/umair-rao" target="_blank"><img src="images/github.svg" alt="Github logo" /></a></li>
+                <li><a href=""><img src="images/angellist.svg" alt="Angellist logo" /></a></li>
             </ul>
             </div>
             <div id="action">
-                <img src="images/Action.svg" alt="Action-icon">
+                <img src="images/Action.svg" alt="">
```
Comment (Qoosim) [inline index.html]: - [ ] Kindly add alt text for this `` tag for the accessibility purpose.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @umair-rao, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-   Accessibility test performed. :heavy_check_mark: 
-   Good GitHub Flow. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] While trying to run color contrast on your page, there's a portion of your page that failed the test. Take a look at the screenshot below. You can use this [link](https://www.aditus.io/button-contrast-checker/) to test it and make changes where necessary. 

color-contrast

File: index.html (hunk 1 above)
Comment (umair-rao) [reply index.html]: <h1> tag added as suggested.

File: index.html (hunk 2 above)
Comment (umair-rao) [reply index.html]: Alt text added as suggested.

**Comment by umair-rao** [comment]
Contrast is added for the button but the test is still failing and asking to change buttons colors which are added as per the template. 

The last reviewer didn't advise any color for buttons or instructions on whether can we change the color of the buttons from the template or not.

**Comment by Bondok6** [comment]
## Hello @umair-rao 🙋‍♂️,

I suggest using color tone is close to the original, for example you can use `#287e5a` instead of  `#36b37e`.
But for the hard work you've done, I'll mark this change as optional, and I am sure that you will fix this minor issue.

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @umair-rao 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [ ] check my comment above ⬆

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**